cli/asc.js

```diff
@@ -270,6 +271,73 @@ exports.main = function main(argv, options, callback) {
 
   // Set up base directory
   const baseDir = args.baseDir ? path.resolve(args.baseDir) : ".";
+  const target = args.target;
+
+  // Once the baseDir is calculated, we can resolve the config, and its extensions
+  let asconfig = getAsconfig(args.config, baseDir, readFile);
+  let asconfigDir = baseDir;
+
+  const seenAsconfig = new Set();
+  seenAsconfig.add(path.join(baseDir, args.config));
+
+  while (asconfig) {
+    // merge target first, then merge options, then merge extended asconfigs
+    if (asconfig.targets && asconfig.targets[target]) {
+      args = optionsUtil.merge(exports.options, args, asconfig.targets[target]);
+    }
+    if (asconfig.options) {
+      args = optionsUtil.merge(exports.options, args, asconfig.options);
+    }
+
+    // entries are added to the compilation
+    if (asconfig.entries) {
```

Comment:
There could also be a workspace field, which could each have its own `ascofig.json.`

Reply:
Or should this been it's own higher level cli like cargo?